docs/whats-new.asciidoc

```diff
@@ -4,165 +4,154 @@
+[role="screenshot"]
+image::whats-new/images/8.5/affects-rule.png[Shows how many rules are affected by an exception]
 
 *New prebuilt rules*
 
-18 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added in 8.4.0.
+28 new {security-guide}/prebuilt-rules.html[prebuilt rules] were added in 8.5.0. 
 
-*Prerequisites and setup guides for Elastic prebuilt rules*
-
-Elastic prebuilt rules now provide additional information to help you identify and meet their {security-guide}/rules-ui-management.html#rule-prerequisites[prerequisites]. You can confirm these requirements in the *Related integrations* and *Required fields* sections of a rule's details page, and consult its *Setup guide* for additional guidance.
+[discrete]
+== Endpoint response enhancements 
 
-[role="screenshot"]
-image::whats-new/images/8.4/rule-details-prerequisites.png[Rule details page with Related integrations, Required fields, and Setup guide highlighted]
+*Updated messaging in response console*
 
-NOTE: Content for these new sections is delivered in a prebuilt rules update, independent of {stack} release versioning.
+The {security-guide}/response-actions.html[response console] UI now displays a message if response action commands aren't supported by an installed version of {agent}, which must be 8.4 or later. 
 
-[discrete]
-== Response console for endpoint response actions
+*Response actions history log for all endpoints*
 
-The new response console allows you to perform response actions on an endpoint using a terminal-like interface. You can enter action commands and receive almost immediate feedback. Actions are also logged in the endpoint’s actions log for reference.
+You can now filter and search endpoint response actions history for an endpoint. In addition, there's now a standalone {security-guide}/response-actions-history.html[Response actions history] page that shows an action log of all endpoints.
 
 [role="screenshot"]
-image::whats-new/images/8.4/response-console.png[Response console]
+image::whats-new/images/8.5/response-history.png[Response history action log]
 
 [discrete]
-== Troubleshooting "Unhealthy" status for {agent}
+== New Entity Analytics dashboard 
 
-Integration policy errors and statuses are now provided in {fleet} and {elastic-sec} to help troubleshoot when an {agent} has an "unhealthy" status.
+The {security-guide}/detection-entity-dashboard.html[Entity Analytics dashboard] provides a centralized view of emerging insider threats - including host risk, user risk, and notable anomalies from within your network. Use it to triage, investigate, and respond to emerging threats. You can also enable host risk score and user risk score directly from this dashboard by clicking the *Enable* button. To learn more, refer to {security-guide}/host-risk-score.html#enable-host-risk-score[Host risk score] and {security-guide}/user-risk-score.html#deploy-user-risk-score[User risk score].
```

Comment:
Hard to tell in GitHub, but changing this to an em dash.

Also, would it be helpful to explain how much easier the **Enable** button method is, compared to the previous method? That seems like a big win for customers. It might help to break that feature into a separate paragraph, maybe add some extra wording -- but please scale this back if it sounds too sales-pitchy (or if "no other setup required" isn't actually accurate):
```suggestion
The {security-guide}/detection-entity-dashboard.html[Entity Analytics dashboard] provides a centralized view of emerging insider threats — including host risk, user risk, and notable anomalies from within your network. Use it to triage, investigate, and respond to emerging threats. 

You can also enable host risk score and user risk score directly from this dashboard simply by clicking the *Enable* button; no other setup is required. To learn more, refer to {security-guide}/host-risk-score.html#enable-host-risk-score[Host risk score] and {security-guide}/user-risk-score.html#deploy-user-risk-score[User risk score].
```